ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py

```diff
@@ -172,3 +172,36 @@ def test_sparse_categorical_crossentropy(
         y_pred=y_pred,
         from_logits=from_logits,
     )
+
+
+# mean_absolute_error
+@handle_cmd_line_args
+@given(
+    y_true=st.lists(st.integers(min_value=0, max_value=4), min_size=1, max_size=1),
+    dtype_y_pred=helpers.dtype_and_values(
+        available_dtypes=ivy_tf.valid_float_dtypes,
+        shape=(5,),
+        min_value=-10,
+        max_value=10,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.mean_absolute_error"
+    ),
+)
+def test_mean_absolute_error(
+        y_true, dtype_y_pred, as_variable, num_positional_args, native_array, fw):
+    y_true = ivy.array(y_true, dtype=ivy.int32)
+    dtype, y_pred = dtype_y_pred
+
+    helpers.test_frontend_function(
+        input_dtypes=[ivy.int32, dtype],
+        as_variable_flags=as_variable,
+        with_out=False,

```

Comment:
with_out flag should be passed in as an argument of test_mean_absolute_error as it is handle by the handle_cmd_line_args as well.

Reply:
you should have a with_out argument for test_mean_absolute_error and let with_out=with_out here.